### PR TITLE
Non-descriptive heading amends for accessibility (AA)

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3506

### Change description ###
Non-descriptive heading amends for accessibility (AA)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/shared/components/case-editor/case-edit-page/case-edit-page.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
-<!-- Current Page && Event trigger name -->
-<h1 class="heading-h1"><span *ngIf="currentPage && currentPage.label">{{currentPage.label}}&nbsp;-&nbsp;</span>{{ eventTrigger.name}}</h1>
+<ng-container *ngIf="currentPage">
+  <h1 *ngIf="!currentPage.label" class="govuk-heading-l">{{eventTrigger.name}}</h1>
+  <ng-container *ngIf="currentPage.label">
+    <span class="govuk-caption-l">{{ eventTrigger.name}}</span>
```
Comment (paul-graham) [inline src/shared/components/case-editor/case-edit-page/case-edit-page.html:4]: These look to be in a different order to the original of `currentPage.label` followed by `eventTrigger.name` (and also separated by a hyphen). Is the change in order and lack of a hyphen intentional?

**Review by paul-graham — CHANGES_REQUESTED** [review]
Just the one thing to highlight...

File: src/shared/components/case-editor/case-edit-page/case-edit-page.html (hunk 1 above)
Comment (paulhowes-HMCTS) [reply src/shared/components/case-editor/case-edit-page/case-edit-page.html:4]: Yes - it's basically being split in two. If there is no label then the eventTrigger is the heading, and if there is a label the label becomes the heading and the eventTrigger becomes the gov-uk caption that sits above the heading.

**Review by paul-graham — APPROVED** [review]
Given the response to my question, I'm happy that everything here looks fine. 👍

**Review by phillip-whitaker — APPROVED** [review]

**Pull request merged**